inc/blocks/namespace.php

```diff
@@ -157,6 +159,86 @@ function on_save_post( int $post_ID, WP_Post $post, bool $update ) : void {
 	}
 }
 
+/**
+ * Synchronise any XBs on the widget page with a shadow post type.
+ *
+ * @param instance $instance Widget Object.
+ * @param new_instance $new_instance Widget Object.
+ * @param old_instance $old_instance Widget Object
+ * @param widget $widget Current Widget Instance
+ * @return object
+ */
+function on_widgets_save( $instance, $new_instance, $old_instance, $widget ) {
+
```

Comment:
Occurred to me that we're relying on `$instance` or `$new_instance` having the `content` key, for backwards compatibility can you add a check to return early is `! isset( $instance['content'] )`? Not sure if there's a more robust way to check if this is a block widget or not.